go.mod

```diff
@@ -75,3 +75,5 @@ require (
 replace github.com/h2non/gock => gopkg.in/h2non/gock.v1 v1.0.14
 
 go 1.13
+
+replace github.com/drone/go-scm => github.com/quantonganh/go-scm v1.8.1-0.20210107022054-cb9eef77a93f
```

Comment:
This should be removed prior to merging (but after blocking PR is merged)

Reply:
Of course. It's temporary to make the test passed.